open-apparel-registry/open-apparel-registry
# Create more realistic fixtures and use real matching

## Overview

Replaces random data in fixtures with more reasonable looking data and removes the use of facility and facilitymatch fixtures in favor of using the real match process.

- Adds CSVs with geocoded results
- Processor loads geocoded points from the CSV rather than making geocode requests
- Makes contributor names based on type
- Makes all fixtures lists active
- Make `cN@example.com` the username so that we can easily log in as any of the fixture users.
- Makes more realistic randomized file names rather then using person names.
- Adds a command to run all the processing steps on all the fixture data.

Connects #301 

## Demo

<img width="1416" alt="Screen Shot 2019-03-14 at 3 35 33 PM" src="https://user-images.githubusercontent.com/17363/54396191-6a491780-466f-11e9-9053-0ed3827a8561.png">

<img width="474" alt="Screen Shot 2019-03-14 at 3 39 41 PM" src="https://user-images.githubusercontent.com/17363/54396201-703ef880-466f-11e9-9420-6aa85a068728.png">


## Testing Instructions

* Verify `./scripts/manage resetdb` runs without error
* Verify `./scripts/manage processfixtures` runs without error 
* Log on as `c8@example.com`:`password`. Browse http://localhost:6543/lists/8 and verify that there are potential matches.
